floris/simulation/wake_deflection/base_velocity_deflection.py

```diff
@@ -189,18 +189,155 @@ def calculate_effective_yaw_angle(
             tmp = avg_V - np.mean(Veff, axis=1)
             target_yaw_ix = np.argmin(np.abs(tmp))
 
-            if target_yaw_ix is not None:
+            if flow_field.yaw_eff is False:
+                return turbine.yaw_angle
+            elif target_yaw_ix is not None:
                 yaw_effective = test_gamma[target_yaw_ix]
             else:
                 err_msg = "No effective yaw angle is found. Set to 0."
                 self.logger.warning(err_msg, stack_info=True)
                 yaw_effective = 0.0
 
+            turbine.effective_yaw = yaw_effective
+
             return yaw_effective + turbine.yaw_angle
 
         else:
             return turbine.yaw_angle
 
+    def calculate_effective_tilt_angle(
+        self, x_locations, y_locations, z_locations, turbine, coord, flow_field
+    ):
+        """
+        This method determines the effective yaw angle to be used when
```

Comment:
This doc string references "yaw angle", but it should refer to "tilt angle". Is the paper listed here relevant to tilt, as well?

Reply:
Yes, I think it will be the same paper

Reply:
Ok think this is fixed now